### PR TITLE
Add ConveRTTokenizer

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Add `ConveRTTokenizer`
- `ConveRTFeaturizer` can now be used with `return_sequence` set to `True`
- Use `ConveRTTokenizer` as tokenizer in the pipeline `pretrained_embeddings_convert`

closes https://github.com/RasaHQ/rasa/issues/4978

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py [hunk 1]
```diff
@@ -72,19 +70,93 @@ def _compute_features(
...
+    @staticmethod
+    def _tokens_to_text(list_of_tokens: List[List[Token]]) -> List[Text]:
+        texts = []
+        for sent_tokens in list_of_tokens:
+            text = ""
+            offset = 0
+            for token in sent_tokens:
+                if offset != token.start:
+                    text += " "
+                text += token.text.replace("﹏", "")
```
Comment (dakshvar22) [inline rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py]: Can you add a comment here about where do these special characters come from?

Comment (dakshvar22) [inline rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py]: Would it be possible to perform this post-processing on tokens inside the tokenizer itself?
Would it mess up the entity offset annotations?

**Comment by dakshvar22** [comment]
@tabergma Can you also rename `rasa/nlu/featurizers/featurzier.py` to `rasa/nlu/featurizers/featurizer.py` :)

**Comment by tabergma** [comment]
We need to add more functionality. Will open a new PR once implementation is done.

**Pull request closed without merging**